web-registry/src/components/organisms/RegistryLayout/components/ConnectWalletModal/ConnectWalletModal.tsx

```diff
@@ -0,0 +1,35 @@
+import Modal, { RegenModalProps } from 'web-components/lib/components/modal';
+import ConnectWallet from 'web-components/lib/components/organisms/ConnectWallet';
+
+import { Link } from 'components/atoms';
+import { WalletButton } from 'components/organisms/WalletButton/WalletButton';
+
+import {
+  CONNECT_WALLET_MODAL_DESCRIPTION,
+  CONNECT_WALLET_MODAL_HREF,
+  CONNECT_WALLET_MODAL_LINK,
+  CONNECT_WALLET_MODAL_TITLE,
+} from './ConnectWalletModal.constants';
+
+interface Props extends RegenModalProps {}
+
+export const ConnectWalletModal = ({ open, onClose }: Props) => (
```

Comment:
Why not just use `RegenModalProps` directly?
```suggestion
export const ConnectWalletModal = ({ open, onClose }: RegenModalProps) => (
```

Reply:
So it's easier to add props later to this modal if needed.